components/automate-ui/src/app/modules/cds/enable-content-modal/enable-content-modal.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, Input, Output, SimpleChanges, OnChanges, EventEmitter } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-enable-content-modal',
+  templateUrl: './enable-content-modal.component.html',
+  styleUrls: ['./enable-content-modal.component.scss']
+})
+export class EnableContentModalComponent implements OnChanges {
+  @Input() show: boolean;
+  @Output() onTokenSubmit: EventEmitter<string> = new EventEmitter();
+
+  public modalVisible = true;
+  public modalLocked = false;
+  public tokenForm: FormGroup;
+
+  constructor(
+    public fb: FormBuilder,
+    private router: Router
+  ) {
+    this.tokenForm = fb.group({
+      token: ['', [Validators.required]]
+    });
+  }
+
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes['show']) {
+      this.modalVisible = changes.show.currentValue;
+    }
+  }
+
+  public closeModal(): void {
+    this.router.navigate(['/']);
```

Comment:
This allows a user to navagate away from the CDS page if they do not have a token.